### PR TITLE
Day6

### PR DESCRIPTION
**Pull request by fstetlerKnightec**

### CODE REVIEW

File: src/main/java/org/advent/day6/TimeDistance.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package org.advent.day6;
+
+public class TimeDistance {
+
+    private int timeRaceLasts;
+
+    private int totalDistanceTraveled;
+
+    public TimeDistance() {
```
Comment (Thomas-Zangerl) [inline src/main/java/org/advent/day6/TimeDistance.java:9]: delete, unused

File: src/main/java/org/advent/day6/TimeDistance.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        this.totalDistanceTraveled = distance;
+    }
+
+    public int getTimeRaceLasts() {
+        return timeRaceLasts;
+    }
+
+    public void setTimeRaceLasts(int timeRaceLasts) {
+        this.timeRaceLasts = timeRaceLasts;
+    }
```
Comment (Thomas-Zangerl) [inline src/main/java/org/advent/day6/TimeDistance.java:24]: delete, unused

File: src/main/java/org/advent/day6/TimeDistance.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+    public int getTotalDistanceTraveled() {
+        return totalDistanceTraveled;
+    }
+
+    public void setTotalDistanceTraveled(int totalDistanceTraveled) {
+        this.totalDistanceTraveled = totalDistanceTraveled;
+    }
```
Comment (Thomas-Zangerl) [inline src/main/java/org/advent/day6/TimeDistance.java:32]: delete, unused

File: src/main/java/org/advent/day6/DaySix.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+        int totalTimeDuration = timeDistance.getTimeRaceLasts();
+        for (int buttonTime = 0; buttonTime < totalTimeDuration; buttonTime++) {
+            int distanceTraveled = distanceTraveledPerRace(buttonTime, totalTimeDuration);
+            if (timeDistance.getTotalDistanceTraveled() < distanceTraveled) {
+                totalNumberOfSettingsAbleToBeatRecord += 1;
+            }
+        }
+        return totalNumberOfSettingsAbleToBeatRecord;
+    }
```
Comment (Thomas-Zangerl) [inline src/main/java/org/advent/day6/DaySix.java:23]: this method should be on the class

File: src/main/java/org/advent/day6/DaySix.java [hunk 5]
```diff
@@ -0,0 +1,64 @@
+package org.advent.day6;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DaySix {
+
+    public int distanceTraveledPerRace(int buttonTime, int totalTimeDuration) {
+        return buttonTime*(totalTimeDuration - buttonTime);
+    }
```
Comment (Thomas-Zangerl) [inline src/main/java/org/advent/day6/DaySix.java:10]: this should also not be here

Comment (fstetlerKnightec) [reply src/main/java/org/advent/day6/DaySix.java:10]: Fixed

File: src/main/java/org/advent/day6/DaySix.java (hunk 4 above)
Comment (fstetlerKnightec) [reply src/main/java/org/advent/day6/DaySix.java:23]: Done

File: src/main/java/org/advent/day6/TimeDistance.java (hunk 1 above)
Comment (fstetlerKnightec) [reply src/main/java/org/advent/day6/TimeDistance.java:9]: Dione

File: src/main/java/org/advent/day6/TimeDistance.java (hunk 2 above)
Comment (fstetlerKnightec) [reply src/main/java/org/advent/day6/TimeDistance.java:24]: Done

**Pull request closed without merging**